tests/kserve/sklearn-iris-private.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-iris-private"
```

Comment:
why do we have the -private suffix? what is the justified benefit of having it ?